src/components/Network.vue

```diff
@@ -53,15 +50,15 @@ class Network extends Vue {
             || keyguardRequest.recipientType !== Nimiq.Account.Type.BASIC
         ) {
             tx = new Nimiq.ExtendedTransaction(
-                new Nimiq.Address(new Nimiq.SerialBuffer(keyguardRequest.sender)),
+                new Nimiq.Address(keyguardRequest.sender),
```

Comment:
Oh cool, I didn't know the Nimiq primitives accept generic `Uint8Arrays`.
I somehow always thought they expect `SerialBuffer`s. :+1:

Reply:
I thought that too back then, that's why it used serialbuffers nearly everywhere.

Turns out, the constructor accepts (and actually needs) an uint8array and the `unserialize` methods require a serialbuffer.

Reply:
`SerialBuffer` extends `Uint8Array`, thus it was also ok before, just not necessary.

Reply:
Yes, it _works_. But internally, the primitives use Uint8Arrays, and when constructing one with a SerialBuffer, it assigns it to `this._obj`, which then throws a type-mismatch in tests :D

So using Uint8Arrays in the constructor is the _correct_ way.